pkgs/development/python-modules/mitmproxy/default.nix

```diff
@@ -136,6 +137,6 @@ buildPythonPackage rec {
     homepage = "https://mitmproxy.org/";
     changelog = "https://github.com/mitmproxy/mitmproxy/blob/${version}/CHANGELOG.md";
     license = licenses.mit;
```

Comment:
Could you add sourceProvenance please ?

Reply:
Should we set anything other than source which is the implicit default?

Reply:
See here, but it depends on the package and the way it is built, but on this one I believe it is `fromSource` : 
https://nixos.org/manual/nixpkgs/unstable/#sec-meta-sourceProvenance

I also prefer if it is explicitly set than implicitly.

And I also believe this might be relevant : 
> The presence of a particular sourceType in a package’s meta.sourceProvenance list indicates that the package contains some components falling into that category, though the absence of that sourceType does not guarantee the absence of that category of sourceType in the package’s contents. A package with no meta.sourceProvenance set implies it has no known sourceTypes other than fromSource.